ui/packages/tidb-dashboard-for-clinic-op/src/apps/SlowQuery/context.ts

```diff
@@ -119,6 +131,7 @@ export const ctx: (extra: DsExtra) => ISlowQueryContext = (extra) => ({
     apiPathBase: client.getBasePath(),
     enableExport: false,
     showDBFilter: false,
-    showDigestFilter: false
+    showDigestFilter: false,
+    showResourceGroupFilter: true
```

Comment:
For clinic-op, I feel `showResourceGroupFilter` is better to `false`, although in this case, `true` and `false` are the same result.

Reply:
What do clinic-op and clinic-could stand for? dedicated and serverless?

Reply:
clinic-op for the OP clusters, they can collect the diagnosis data by tiup diag tool in local and upload to clinic server, while clinic-cloud for the cloud clusters.